### PR TITLE
Jdcheatham

### PR DESCRIPTION
**Pull request by jdcheatham**
last part.  Just want to make this work.  Realizing I'm probably shooting myself in the foot a bit trying to run everything from a linux virtual machine, then again, where's the fun in changing things now?

### CODE REVIEW

File: jdcheatham/Lab4.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""J. Cheatham
+CPSC4500, lab4"""
+
+
+def verifyDecrease(number):
+    if number[0] < number[1]:
```
Comment (ktarplee) [inline jdcheatham/Lab4.py:9]: add docstrings

File: jdcheatham/Lab4.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return ("false")
+    else:
+        if number[0] < number[2]:
+            return ("false")
+    return "true"
+
+
+def magic(number):
+    numCheck = str(number)
```
Comment (ktarplee) [inline jdcheatham/Lab4.py:21]: add docstrings

File: jdcheatham/Lab4.py (hunk 1 above)
Comment (jdcheatham) [inline jdcheatham/Lab4.py:9]: Doc strings added.

**Comment by ktarplee** [comment]
This pull request looks much better from a content perspective than your new pull request.  Just make it pass the tests now.

**Pull request merged**